docker-compose.yml

```diff
@@ -20,6 +19,12 @@ services:
     ports:
       - "16379:6379"
 
+  redis74:
+    image: redis:7.4
```

Comment:
Is there anything that sets redis 7.4 apart, feature-wise, that warrants adding an additional service container for tests *AND* keeping 7.2? 

If not I'd rather bump the existing redis 7.2 to 7.4 and use that.

Reply:
`watch` was added in Redis 7.4 and in ValKey 8.0. The latter also added `tot-net-in`, `tot-net-out`, `tot-cmds`. It seems ValKey != Redis (at least in versions).
But for `watch` we can use Valkey 8.0.

Reply:
Yes, there there is at least one field which is Redis 7.4 specific. There are a lot more info fields - but that's another PR.

ValKey 7.2.5 is the latest ValKey 7.x release and is compatible with Redis 7.2.4 . In fact he just released ValKey 8.0 seems to include some Redis 7.4 changes but its officially compatible with Redis 7.2.4. It seems that Redis & ValKey can no longer be treated the same.

Reply:
> There are a lot more info fields - but that's another PR.

Great - looking forward to it!

> It seems that Redis & ValKey can no longer be treated the same.

Yeah, unfortunately.

Do you think it makes sense to bump the dependencies to the latest valkey and Redis versions?

Reply:
Made a simple test of using valkey 8..0 - test run failed with many errors. It seems extended compatibility mode is needed https://github.com/valkey-io/valkey/pull/306 .
It seems like beyond the scope of this PR?

Reply:
Agreed. Looks like we need some dedicated effort to make this work with valkey